site/content/how-to-monetize-apps/monetization.md

```diff
@@ -0,0 +1,92 @@
+---
+weight: 10 
+---
+
+# How to monetize apps?
+
+<!--Abstract, short intro -->
+Whether your app is already up and running or you're just starting off, the topic of **app monetization** will eventually cross your mind. If you don't know how to go about generating income from your apps, this is the place for you to start.
+
+
+<!-- Intro for newcomers -->
+## Introduction
+If you're new to LiveChat, here's how things work in here. LiveChat can be extended with various applications built by developers. All of this is done to meet the specific business needs of those who use LiveChat. You, as a developer and app author, can sell it on our [Marketplace](https://www.livechatinc.com/marketplace/). 
+
+If you'd like to know what types of applications can be built upon LiveChat Platform, check out the [Building LiveChat apps](/building-apps-for-livechat/) doc. 
+
+The place to start creating your apps with different building blocks is our [Developer Console](https://developers.livechatinc.com/console/). If your app is ready, you can continue with the monetization process. 
+
+<!-- How the process looks -->
+## Core information
+
+Here's the core info about monetization at **LiveChat**:
+
+* You can perform all the actions in one place- [Developer Console](https://developers.livechatinc.com/console/), which speeds up the whole procedure. 
+* We use PayPal as our billing platform. 
+* The provision is 20%, so if you set the app price to be 10$, you will earn 8$ net. 
+* We offer two billing cycles: **once a month** and **one-time payment**. We recommend the recurring payment option. We're currently working on the implementation on **micropayments**, so stay tuned for this one!
```

Comment:
>* We offer two billing cycles: **once a month** and **one-time payment**. We recommend the recurring payment option. ~We're currently working on the implementation on **micropayments**, so stay tuned for this one!~

I'd put it that way:

>* We offer two billing cycles: **once a month** and **one-time payment**. We recommend the recurring payment option, as LiveChat itself is a SaaS solution.
>* You can leverage LiveChat [Billing API](/docs/billing-api/) to implement **micropayments** in your app.

Reply:
The only problem is that this link you entered to Billing API on GitHub doesn't work. I linked to Docs instead. https://developers.livechatinc.com/docs/billing-api/